mocks/routes/thc.js

```diff
@@ -0,0 +1,132 @@
+// Use this file only as a guide for first steps using routes. Delete it when you have added your own route files.
+// For a detailed explanation regarding each routes property, visit:
+// https://mocks-server.org/docs/usage/routes
+
+const humidities = [
+  { timestamp: 1, humidity: 30 },
+  { timestamp: 2, humidity: 40 },
+  { timestamp: 3, humidity: 50 },
+  { timestamp: 4, humidity: 42 },
+  { timestamp: 5, humidity: 38 },
+]
+
+const temperatures = [
+  { timestamp: 1, temperature: 10.1 },
+  { timestamp: 2, temperature: 12.3 },
+  { timestamp: 3, temperature: 14.8 },
+  { timestamp: 4, temperature: 27.2 },
+  { timestamp: 5, temperature: 29.2 },
+]
+
+const co2s = [
+  { timestamp: 1, co2: 36 },
+  { timestamp: 2, co2: 81 },
+  { timestamp: 3, co2: 30 },
+  { timestamp: 4, co2: 64 },
+  { timestamp: 5, co2: 76 },
+]
+
+module.exports = [
+  {
+    id: "get-temperature", // route id
+    url: "/api/temperature", // url in express format
+    method: "GET", // HTTP method
+    variants: [
+      {
+        id: "success",
+        type: "middleware", // variant of type "middleware"
+        options: {
+          middleware: (req, res) => {
+            const query = req.query 
+            if (query.current==true) {
```

Comment:
Checking true like this does not return the expected result. Because query.current is a string "true" the check should be done like below.

```suggestion
            if (query.current === "true") {
```